geth/jail/jail_test.go

```diff
@@ -115,6 +128,46 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+
+	defer func() {
+		if err := recover(); err != nil {
+			require.NotNil(err, "Panic occured")
+		}
+	}()
+
+	require.NotNil(s.jail)
```

Comment:
No need to check that. It's responsibility of Setup code to ensure it's not nil.